### PR TITLE
Fix upstream UI changes.

### PR DESCRIPTION
**Pull request by amavinag**
* change the dropdown menu element.
  * make ui changes inline with 5.6 latest version.

### CODE REVIEW

**Comment by amavinag** [comment]
test_results: https://gist.github.com/amavinag/14e7687b4394a9c2282fa0865f8d2a1e

**Comment by dajoRH** [comment]
Lint report for commit a72b0ddc0a230f01d3784fa8df38ffc4e0150221:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/dashboard.py [hunk 1]
```diff
@@ -24,8 +24,7 @@
             '5.4': '//nav//ul[contains(@class, "navbar-utility")]'
                    '/li[contains(@class, "dropdown")]/a',
             '5.6.0.1': '//nav//a[@id="dropdownMenu2"]',
-            version.LATEST: '//nav//ul[contains(@class, "navbar-utility")]'
```
Comment (psav) [inline cfme/dashboard.py]: You can delete version.LATEST completely

**Comment by dajoRH** [comment]
Lint report for commit d34348c8bdf7b9f55efa169baaebf37830b5a5bb:

Everything seems all right :smile: .
*CFME QE Bot*

**Pull request merged**